### PR TITLE
A bare bones .travis.yml

### PR DESCRIPTION
**Pull request by alexeyklyukin**
Per #6

### CODE REVIEW

**Comment by alexeyklyukin** [comment]
👍

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+before_install:
+  - go get github.com/Masterminds/glide
+  - go get github.com/mattn/goveralls
+
+install:
+  - make deps
+
+script:
+  - goverals -service=travis=ci
```
Comment (CyberDem0n) [inline .travis.yml]: `-service=travis=ci`

Looks fishy to me. Shouldn't it be `travis-ci`?

Comment (alexeyklyukin) [inline .travis.yml]: thanks, fixed.

**Comment by alexeyklyukin** [comment]
👍

**Comment by Jan-M** [comment]
:+1:

**Pull request merged**